modules/AWS/asg_ec2_tagging/provider_version.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
```

Comment:
The provider version pin is required in every module. This is to mitigate inconsistency of provider version between root module and child modules during `terraform validate`. Terraform deprecated propagating "version of providers" so this is not clean but a way to align provider version across the project.
See https://www.terraform.io/language/providers/configuration#version-deprecated